net-misc/r8125/files/r8125-9.008.00-linux-5.17.patch

```diff
@@ -0,0 +1,112 @@
+Added compatibility with Linux Kernel 5.17+.
```

Comment:
Please include at the top of the patch:
- link to where it's from / submitted to
- link to relevant Gentoo bug

Reply:
The patch was designed by myself, based on my analysis. There is nowhere to submit it, Realtek doesn't have any public repo. Later I discovered that my patch is almost identical to [the relevant patch for R8168](https://github.com/gentoo/gentoo/commit/13afe46537e99157756939d331bebed49169dfc6), which is a good sign. 

I'll add the info.